app/assets/stylesheets/_search_result.scss

```diff
@@ -55,7 +55,11 @@
     .notes,
     .videos {
       float: left;
+      padding-top: 5px;
       video { margin-bottom: 10px; }
     }
   }
 }
+.search-result-container {
```

Comment:
it looks like this doesnt actually do anything now? Is it still needed for anything?

Reply:
There is a conditional in the checkVocalBarHeight function in the `vocab_bar.js` file that utiltizes the class on line 23 so I should probably leave in.

```
function checkVocabBarHeight() {
      if (barBottom > footerTop) {
        if ($('.search-results__container').length > 0) {
          bar.css({ height: $('.search-results__container').innerHeight() });
        } else if ($('.search-result-container').length > 0) {
          bar.css({ height: $('.search-result-container').innerHeight() });
        }
      }...

```

Reply:
it has a position style on which does not show in this pr but I can move it further up the file

Reply:
Actually I can't move it to what I though was a good place is a specific media style but I CAN extract the style and place in an appropriate place.